common/djangoapps/third_party_auth/urls.py

```diff
@@ -2,11 +2,12 @@
 
 from django.conf.urls import include, patterns, url
 
-from .views import inactive_user_view, saml_metadata_view
+from .views import inactive_user_view, saml_metadata_view, lti_login_view
 
 urlpatterns = patterns(
     '',
     url(r'^auth/inactive', inactive_user_view),
     url(r'^auth/saml/metadata.xml', saml_metadata_view),
+    url(r'^auth/login/(?P<backend>lti)/$', lti_login_view),
```

Comment:
@wmono The way this `lti_login_view` is implemented as a combined login/complete view makes sense to me, but I think what it's doing is much closer to the normal "complete" view than it is to the "login" view. What do you think about using the `auth/complete/lti/` URL instead?